tests/retrieve/test_access.py

```diff
@@ -387,7 +387,7 @@ def test_get_footprint(inlet_keyword, inlet_value):
     assert metadata["min_longitude"] == pytest.approx(float(footprint.lon.min()))
     assert metadata["max_latitude"] == pytest.approx(float(footprint.lat.max()))
     assert metadata["min_latitude"] == pytest.approx(float(footprint.lat.min()))
-    assert metadata["high_time_resolution"] == False
+    assert metadata["high_time_resolution"] == "false"
```

Comment:
What was the issue we had with metadata needing to be saved as strings? I feel like we did look into this but not sure how it got resolved.

If this is a requirement, for searching we'll need to be careful around this as would make data hard to search for using booleans. If not we should make sure this gets saved as a bool.

Reply:
I think outside of the metastore, `search` and the `read_file` functions are calling `clean_string` on all metadata values, so booleans are transformed to lower case strings. If we didn't do this, I think `TinyDB` would store and retrieve booleans.

Reply:
Ok great and looks like we have a test to check this as well - https://github.com/openghg/openghg/blob/Iss561/addZarrStore/tests/retrieve/test_search.py#L343.

Something we may want to look at again in future but seems to work ok for now.